crates/uv-distribution-types/Cargo.toml

```diff
@@ -30,8 +30,10 @@ uv-pypi-types = { workspace = true }
 uv-small-str = { workspace = true }
 
 arcstr = { workspace = true }
+anyhow = { workspace = true }
```

Comment:
Can we make this a dev dependency? (We typically want to avoid using `anyhow` in library crates.)

Reply:
Removed with refactor